src/Components/Endpoints/src/Rendering/EndpointHtmlRenderer.cs

```diff
@@ -135,6 +139,11 @@ internal async Task InitializeStandardComponentServicesAsync(
         }
     }
 
+    private void OnNavigateTo(object? sender, NavigationEventArgs args)
+    {
+        _httpContext.Response.Redirect(args.Uri);
+    }
```

Comment:
I'm not sure this is enough. We need to do something like what we do in the NotFound case, where we stop further renders, as they are not going to be used.

Also, in the case the response has started (streaming rendering), we can't start a redirect this way

Reply:
We can't do the same as in `NotFound` either, that would result in an infinite loop.
1) Allow managing flow by exception only for this case to break the infinite loop. Minus: one edge case will still annoy developers.
2) Inject a refresh `meta http-equiv`  refresh tag into the response. Minus: cannot be cancelled, is implemented on browser level -> the implementation might differ from http redirection.